src/main/java/io/kokuwa/maven/helm/PackageMojo.java

```diff
@@ -116,9 +126,28 @@ public void execute() throws MojoExecutionException {
 			}
 
 			helm.execute("Unable to package chart at " + chartDirectory);
+			setUpPlaceholderFileAsMavenArtifact();
 		}
 	}
 
+	protected void setUpPlaceholderFileAsMavenArtifact() {
```

Comment:
A: please add a comment to describe the purpose the function

Reply:
Done